tests/unit/mainwindow/test_tabwidget.py

```diff
@@ -128,6 +129,22 @@ def test_update_tab_titles_benchmark(self, benchmark, widget,
 
         benchmark(widget.update_tab_titles)
 
+    def test_tab_min_width(self, widget, fake_web_tab, config_stub):
+        """Ensure by default, all tab sizes are the same."""
+        widget.addTab(fake_web_tab(), 'foobar')
+        normal_size = widget.tabBar().minimumTabSizeHint(0).width()
+        normal_size += 100
```

Comment:
IMHO, this is easier to read if you just use `+ 100` in the line above. IMHO, `+=` reads a bit weird if we aren't actually interested in the old value.

Reply:
The reason I split it was because I thought it was more deliberate, ie: 

1. get the current tab size
2. add 100 to it
3. make that value min_width

I was worried combining them would mask the intent, but I guess it worked the other way :P